learn/react-native/download-progress.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Download Progress in Notification"
+id: "download-progress"
+sidebar_label: "Download Progress"
+---
+---
+import DownloadProgress from '/learn/assets/download-progress.png';
+import ProgressBar from '/learn/assets/progress-bar.png';
+
+A download progress bar in a notification is a visual indicator displayed in the notification area of a device to show the progress of a file download. This feature provides users with real-time feedback on the status of their downloads without having to open the app.
+
+Download progress bar in a notification enhances user experience by reducing uncertainty and providing a sense of how much time is remaining for the download to complete. 
+
+When downloading a file, users can use other apps and still monitor the download progress through notifications without needing to open the app.
+
+## Implement Download Progress in Notification in WaveMaker Application
+
+
+### Adding rn-fetch-blob Plugin to your WaveMaker App
+
+- rn-fetch-blob
+
+RNFetchBlob plugin can be installed in a few steps in a WaveMaker application. Please refer to this [page](https://docs.wavemaker.com/learn/react-native/third-party-expo-plugins#expo)
+on how to install the plugin. 
+
+<img src={DownloadProgress} style={{width:"35%"}} />
+
+
+<img src={ProgressBar} style={{width:"35%"}} />
+
+The following is the code snippet for the above mockup
+
+```javascript
+
+const RNFetchBlob = require('rn-fetch-blob').default;
+
+
+Page.downloadfileTap = function($event, widget) {
+
+    const image = Page.Widgets.urlinput.datavalue;
+    let dirs = RNFetchBlob.fs.dirs;
```

Comment:
Does this work in web-preview?
What happens if the page is opened in web-preview.
Document that also.